migrator/src/main/resources/migrationJob.json

```diff
@@ -60,16 +41,16 @@
     "schema": {
         "access": {
             "delete": [
-                "lb-migrationjob-delete"
+                "anyone"
             ],
             "find": [
-                "lb-migrationjob-find"
+                "anyone"
             ],
             "insert": [
-                "lb-migrationjob-insert"
+                "anyone"
             ],
             "update": [
-                "lb-migrationjob-update"
+                "anyone"
```

Comment:
Did that so I can test. Will revert later.

On Wed, Jun 3, 2015 at 8:10 AM, Dennis Crissman <notifications@github.com>
wrote:

> In migrator/src/main/resources/migrationJob.json
> <https://github.com/lightblue-platform/lightblue-migrator/pull/197#discussion_r31625894>
> :
>
> >              ],
> >              "update": [
> > -                "lb-migrationjob-update"
> > +                "anyone"
>
> Do we want anyone to be able to make changes to migrationJob?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/197/files#r31625894>
> .
>